torch/lib/c10d/reducer.cpp

```diff
@@ -53,6 +54,21 @@ Reducer::Reducer(
   TORCH_CHECK(replicas_.size() >= 1, "Expected at least one model replica.");
   TORCH_CHECK(replicas_[0].size() >= 1, "Expected at least one parameter.");
 
+  // Check whether the module is multi_device_module
+  {
```

Comment:
nit: do we need this extra pair of brackets?

Reply:
following up code style in this constructor:)